docs/installation.md

```diff
@@ -19,6 +19,57 @@ spack install additivefoam
 ```
 spack `develop` is currently required.
 
+### Spack installation with external modules
+
+If you are installing OpenFOAM through spack, some configuration is needed to enable spack to use external modules, which is common on HPC systems. For OpenFOAM, this is largely needed for the C++ compiler and MPI dependencies, which are both generally available as preconfigured modules that can be loaded for functionality that has been configured for a specific HPC system. While module versions and names may vary from system to system, this example provides a template for configuring spack on your system.
```

Comment:
I would revise to say this example will use a standard HPC module setup, but the general idea of using an existing compiler/MPI can work anywhere